thunder/tests/test_inplace_functionalization.py

```diff
@@ -549,6 +549,7 @@ def single_tensor_adam(
     ref_state_steps = [torch.tensor(1, device=device) for _ in range(2)]
     single_tensor_adam(*ref_tensors, state_steps=ref_state_steps)
 
+    # torch.compile does not support accessing the ContextVariable compile data used in _copy__impl_
```

Comment:
This comment is outdated as `_copy__impl_` doesn't access the compile data, could you please remove it?